build.gradle.kts

```diff
@@ -225,6 +224,35 @@ swaggerSources {
     }
 }
 
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    // limit format enforcement to just the files changed by this branch
+    ratchetFrom("origin/main")
+
+    format("misc") {
+        target("*.gradle", ".gitattributes", ".gitignore", ".containerignore", ".dockerignore")
+
+        trimTrailingWhitespace()
+        indentWithSpaces()
+        endWithNewline()
+    }
+    java {
+        targetExclude("**/src/generated/**", "**/build/generated/**")
+
+        googleJavaFormat()
+
+        removeUnusedImports()
+        importOrder("java", "javax", "jakarta", "com", "org", "net.atos")
+
+        formatAnnotations()
+        licenseHeader("""
+/*
```

Comment:
Does this mean all our Java license headers need to be like this?

Reply:
Yes. I believe we have the option to specify different targets and license headers for them, but that might make the config pretty bloated.

Reply:
Ok. I think it's probably a good idea anyway to have them all the same. :-)